flake.nix

```diff
@@ -4,7 +4,7 @@
     nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
     treefmt-nix.url = "github:numtide/treefmt-nix";
     treefmt-nix.inputs.nixpkgs.follows = "nixpkgs";
-    crane.url = "github:ipetkov/crane";
+    crane.url = "github:ipetkov/crane?ref=v0.18.0";
```

Comment:
Maybe we can add a comment here or a TODO to remove the fixed version of `crane`?